packages/wallet-sdk/src/sign/walletlink/WalletLinkSigner.ts

```diff
@@ -451,7 +407,6 @@ export class WalletLinkSigner implements Signer {
   }
 
   private _personal_sign(params: RequestParam) {
-    this._requireAuthorization();
```

Comment:
no need to call `_requireAuthorization`
it's handled on the provider level before hitting WLSigner